### PR TITLE
Issue 121

### PR DESCRIPTION
**Pull request by ckouder**

### CODE REVIEW

File: env/Caddyfile [hunk 1]
```diff
@@ -1,18 +0,0 @@
-hub.cyberexperimentation.org {
```
Comment (carboxylman) [inline env/Caddyfile]: Seems like maybe this is useful, even if not directly referenced by `docker-compose.yml`.  And this should be in a separate PR I think? It seems unrelated to the issue at hand.

File: env/searcch-frontend-local-dev.env [hunk 2]
```diff
@@ -1,8 +0,0 @@
-# searcch local development frontend keys and secrets
```
Comment (carboxylman) [inline env/searcch-frontend-local-dev.env]: This file needs to stay in the repo.  `docker-compose.yml` references it, and we want local dev envs to "just work" via `docker-compose`, and these are the right env vars to make that happen.

And this should be in a separate PR I think? It seems unrelated to the issue at hand.

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,11263 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (carboxylman) [inline yarn.lock]: Are we moving from npm to yarn? Is there a reason to do that?

Even if so, seems like this needs to be part of some other PR. Seems like we would also want to remove package-lock.json and tell the Dockerfile to use yarn instead of npm?

File: env/searcch-frontend-dev.env [hunk 4]
```diff
@@ -1,7 +0,0 @@
-# searcch development frontend keys and secrets
-
-PRODUCTION=False
```
Comment (carboxylman) [inline env/searcch-frontend-dev.env]: Why are we getting rid of this?  Seems like this is a reasonable starting point if people are deploying their own `-dev` test?  And this should be in a separate PR I think? It seems unrelated to the issue at hand.

File: env/searcch-frontend-prod.env [hunk 5]
```diff
@@ -1,7 +0,0 @@
-# searcch production frontend keys and secrets
```
Comment (carboxylman) [inline env/searcch-frontend-prod.env]: Why are we getting rid of this?  Seems like this is a reasonable starting point if people are deploying their own `-prod` test?  And this should be in a separate PR I think?  It seems unrelated to the issue at hand.

File: store/artifacts.js [hunk 6]
```diff
@@ -102,6 +102,28 @@ export const actions = {
     }
     commit('SET_LOADING', false)
   },
+  async fetchMyArtifacts({ commit }) {
```
Comment (carboxylman) [inline store/artifacts.js:114]: How is this different from `fetchArtifacts`?  Seems like if it is intended to be different, you would want it to use a different `commit` variable.  If you use the existing `SET_ARTIFACTS`, if the user goes to my artifacts then search, won't they see the artifacts fetched from my artifacts?  I think this is what would happen, so you would want to use a separate state var for my artifacts.

**Review by carboxylman — COMMENTED** [review]
@ckouder I made a bunch of substantial comments before I even got to the meat of this commit.  Once you comment or resolve further, I will pull this branch and test the relationship changes.

File: env/Caddyfile (hunk 1 above)
Comment (ckouder) [reply env/Caddyfile]: Sorry @carboxylman I think I made a wrong commit. Let me fix it quickly.

**Review by carboxylman — APPROVED** [review]

**Review by dpemmerich — APPROVED** [review]
